### ISSUE TITLE
Pure JS plugin

### ISSUE THREAD

**Issue by vvarda** [Especializa/nativescript-ng-shadow#1]
Please, make shadow-plugin for Nativescript via pure JS.

**Comment by sushant-j** [comment]
Need something similar for a project I am working on. 
@especializa-user - Any plans on doing this?

**Comment by berardo** [comment]
Yes guys, that's definitely something on my radar but I don't have ETA for the moment.
Any help or suggestion would be highly appreciated :)

**Comment by webleaf** [comment]
Need it too

**Comment by mojimi** [comment]
Very much need this

**Comment by marcelbonfim182** [comment]
Please!!! Very much need this!!!
